### ISSUE TITLE
Table cells are marked as 

### ISSUE THREAD

**Issue by dhivyavk** [ckeditor/ckeditor5#13479]
## 📝 Provide detailed reproduction steps (if any)

In edit mode, add a table. I can see that table cells are marked with role='textbox'
Due to this checking against Accessibility guidelines to be followed we will see the below error. It will be confusing to understand the real purpose of this field with assistive technologies.

**Ensures every ARIA input field has an accessible name**
Rule name
[aria-input-field-name](https://accessibilityinsights.io/info-examples/web/aria-input-field-name)
Success criterion
[WCAG 4.1.2](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
Path
tr:nth-child(1) > td:nth-child(1)
Fix ONE of the following:
aria-label attribute does not exist or is empty.
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty.
Element has no title attribute.

### ✔️ Expected result
Input fields, in this case each table cell should have an accessible name. ARIA input field requires additional markup to ensure it has an accessible name that conveys its purpose to users of assistive technologies.

### ❌ Actual result
Accessible name is missing

### ❓ Possible solution
Solution provided above in Fix ONE of the following section above.

## 📃 Other details

* Browser: …
* OS: …
* First affected CKEditor version: …
* Installed CKEditor plugins: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
Good thing is that `toWidgetEditable()` already supports labels, so it's a matter of providing a proper label. Table caption has the same issue.

[https://github.com/ckeditor/ckeditor5/blob/cc319ca909aa28dcf28475f928d56c1ac7651147/packages/ckeditor5-table/src/tablecaption/tablecaptionediting.ts#L106](https://github.com/ckeditor/ckeditor5/blob/cc319ca909aa28dcf28475f928d56c1ac7651147/packages/ckeditor5-table/src/tablecaption/tablecaptionediting.ts#L106)

Should get a meaningful label.

**Comment by Witoso** [comment]
Improved in the duplicate: #16923
